packages/frontend/core/src/components/blocksuite/block-suite-editor/blocksuite-editor-container.tsx

```diff
@@ -249,22 +234,71 @@ export const BlocksuiteEditorContainer = forwardRef<
         blockId: blockElement.blockId,
       });
       selectManager.set([newSelection]);
-      setScrolled(true);
     };
 
     affineEditorContainerProxy.updateComplete
       .then(() => {
         if (blockElement && !scrolled) {
           handleScrollToBlock(blockElement);
+          setScrolled(true);
         }
-        handleUpdateComplete();
       })
       .catch(console.error);
+  }, [
+    blockElement,
+    affineEditorContainerProxy,
+    mode,
+    scrolled,
+    currentPath,
+    locationHash,
+  ]);
+
+  useEffect(() => {
+    if (currentPath === `/all` || (locationHash && !scrolled)) {
```

Comment:
When will currentPath be '/all'

Reply:
It doesn’t seem to happen now. Before this, in the spilt view, the currentPath would occasionally become all, causing it to be replaced back to the all doc page after clicking on the page.

By the way, when you open a doc in the client, document.location.pathname is always all